torchrl/envs/libs/vmas.py

```diff
@@ -112,7 +112,7 @@ def __init__(
             if "device" in kwargs.keys() and kwargs["device"] != str(env.device):
                 raise TypeError("Env device is different from vmas device")
             kwargs["device"] = str(env.device)
-        super().__init__(**kwargs)
+        super().__init__(**kwargs, allow_done_after_reset=True)
```

Comment:
why is this in this PR?

Reply:
because I have introduced in this PR a flag to allow environments to be done after reset (this is necessary when you have multiple dones). 

vmas needed this flag because it has some scenarios which might happen to be done in some of the batch dimensions after a reset